### PR TITLE
Update API build and generate 0.19 API docs

### PR DESCRIPTION
**Pull request by RichieEscarez**
- remove eventing-contrib (https://github.com/knative/docs/issues/3041)
- update build scripts
- fix serving api build errors (update script)
- build 0.19 API docs for Serving and Eventing

Related: https://github.com/knative/website/pull/240

Staged: https://5fb326fd6fbbcb00e3428a3a--knative.netlify.app/docs/reference/api/

cc/ @vaikas @lionelvillard

### CODE REVIEW

File: docs/reference/api/README.md [hunk 1]
```diff
@@ -82,6 +79,24 @@ To generate a version of the API:
...
 
+   If the script fails, there are a couple possible causes.
+
+   * If you get the
+     `F1116 15:21:23.549503   63473 main.go:129] no API packages found in ./pkg/apis`
+     error, check if a new version of the script is available:
+      https://github.com/ahmetb/gen-crd-api-reference-docs/tags
+
+      The script is kept up-to-date with changes that go into the Kubernetes API.
+      As Knative adds support for those APIs, you might need make sure the
```
Comment (mpetason) [inline docs/reference/api/README.md:90]:
```suggestion
      As Knative adds support for those APIs, you might need to make sure the
```

File: docs/reference/api/README.md [hunk 2]
```diff
@@ -82,6 +79,24 @@ To generate a version of the API:
...
+      corresponding
+      [script `gen-crd-api-reference-docs` version](https://github.com/knative/docs/blob/master/hack/gen-api-reference-docs.sh#L26)
+      is used.
+
+   * If you get the
+     `F0807 13:58:20.621526 168834 main.go:444] type invalid type has kind=Unsupported which is unhandled`
+     error, the import target might have moved. For more details and an example,
```
Comment (mpetason) [inline docs/reference/api/README.md:97]:
```suggestion
     error, the import target might have moved. For more details, and an example,
```
I'm bad at punctuation, maybe the above update looks correct ^

Comment (RichieEscarez) [reply docs/reference/api/README.md:97]: Thanks Mike! Here is the style guide we use for Knative: https://developers.google.com/style/commas

Comment (RichieEscarez) [reply docs/reference/api/README.md]: I've been really busy this week and decided to add the details about the issue i faced this week and quickly dumped details about an issue from the past. I took a second pass at this to better explain. PTAL

**Comment by RichieEscarez** [comment]
@mpetason is you are okay with the text, can you provide the LTGM so that we can merge this and unblock the 0.19 docs release?

**Pull request merged**